docs/src/docs/core/Popper.mdx

```diff
@@ -121,6 +121,13 @@ After each placement change `onPlacementChange` which takes current placement as
 />
 ```
 
+## Portal
+
+By default, Popper content will be rendered inside a [Portal](/core/portal) component.
```

Comment:
Please change links to contain a trailing slash (/core/portal/), it's a small issue with Gatsby – it will not highlight link in navbar if it does not have a trailing slash.

Reply:
Done :)